### PR TITLE
update deps refine some log

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
update pd/tidb/tidb-tools dep
add safe-mode for arbiter 
log logs from sarama for arbiter default

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

**Comment by GregoryIan** [comment]
LGTM

**Comment by 3pointer** [comment]
CI failed, need run `go mod tidy`

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ TEST_DIR := /tmp/tidb_binlog_test
 
 GO       := GO111MODULE=on go
 GOBUILD  := CGO_ENABLED=0 $(GO) build $(BUILD_FLAG)
-GOTEST   := CGO_ENABLED=1 $(GO) test -p 3
+GOTEST   := CGO_ENABLED=1 $(GO) test -p 1
```
Comment (GregoryIan) [inline Makefile]: not parallel, but concurrent

File: arbiter/config.go [hunk 2]
```diff
@@ -94,12 +96,14 @@ func NewConfig() *Config {
 	fs.StringVar(&cfg.Metrics.Addr, "metrics.addr", "", "prometheus pushgateway address, leaves it empty will disable prometheus push")
 	fs.IntVar(&cfg.Metrics.Interval, "metrics.interval", 15, "prometheus client push interval in second, set \"0\" to disable prometheus push")
 	fs.StringVar(&cfg.LogFile, "log-file", "", "log file path")
+	fs.BoolVar(&cfg.OpenSaramaLog, "open-sarama-log", true, "log the logs from sarama (https://github.com/Shopify/sarama), the client of Kafka")
```
Comment (suzaku) [inline arbiter/config.go]:
```suggestion
	fs.BoolVar(&cfg.OpenSaramaLog, "open-sarama-log", true, "save the logs from sarama (https://github.com/Shopify/sarama), a client of Kafka")
```

File: drainer/util.go [hunk 3]
```diff
@@ -58,12 +58,12 @@ func (g *taskGroup) start(name string, f func(), noPanic bool) {
...
 					log.Error("Recovered from panic",
 						zap.Reflect("err", err),
-						zap.Stack("stack"),
+						zap.Stack("real stack"),
 						fName,
 					)
 				}
 			}
-			log.Info("Exit", fName)
+			log.Error("Exit", fName)
```
Comment (suzaku) [inline drainer/util.go]: If `f()` returns successfully, this line is also reached because it's in `defer`, so it's not always an error?

**Pull request merged**